client/client_test.go

```diff
@@ -118,17 +120,18 @@ func TestRegisterAndAddProbe(t *testing.T) {
 
 // TestRemoveProbe tests that the RemoveProbe() method removes a probe from Cloudprober
```

Comment:
Nit: there's no need to comment test functions that have a self-describing name. The assumption on the reader's side is that `TestFoo()` tests `Foo()`.

Reply:
I've changed this accordingly.